api/test/service/upload.test.js

```diff
@@ -492,9 +492,14 @@ async function prepareResources (dynamoClient, s3Client) {
     Limit: options.limit || 30,
     ExpressionAttributeValues: {
       ':u': { S: spaceDid },
-    },  
-    KeyConditionExpression: 'uploaderDID = :u',
-    ProjectionExpression: 'dataCID, carCID, uploadedAt'
+    },
+    // gotta sidestep dynamo reserved words!?
+    ExpressionAttributeNames: {
+      '#space': 'space',
+      '#shard': 'shard'
+    },
```

Comment:
Minor bummer. both `space` and `shard` are reserved keywords in dyanmodb land, so you have to do this aliasing step in queries that want to use them!